app/src/main/AndroidManifest.xml

```diff
@@ -29,6 +29,7 @@
 
     <activity
       android:name=".app.administratorcontrols.AdministratorControlsActivity"
+      android:label="@string/administrator_controls_label"
```

Comment:
This name needs to be updated.

Reply:
Apologies for the mistake. It is corrected in :  https://github.com/oppia/oppia-android/pull/3152/commits/099f9b696aec31a9fc6fc0a3d52808dca6023e2e